File: fudi-nfc/src/main/java/com/romellfudi/fudinfc/util/exceptions/InsufficientCapacityException.java
/*
 * Copyright (c) 2020. BoostTag E.I.R.L. Romell D.Z.
 * All rights reserved
 * porfile.romellfudi.com
 */

package com.romellfudi.fudinfc.util.exceptions;

public class InsufficientCapacityException extends Exception {

    public InsufficientCapacityException() {
    }

    public InsufficientCapacityException(String message) {
        super(message);
    }

    public InsufficientCapacityException(StackTraceElement[] stackTraceElements) {
        setStackTrace(stackTraceElements);
    }
}

File: fudi-nfc/src/main/java/com/romellfudi/fudinfc/util/exceptions/InsufficientCapacityException.kt
/*
 * Copyright (c) 2020. BoostTag E.I.R.L. Romell D.Z.
 * All rights reserved
 * porfile.romellfudi.com
 */
package com.romellfudi.fudinfc.util.exceptions

class InsufficientCapacityException : Exception {
    constructor() {}
    constructor(message: String?) : super(message) {}
    constructor(stackTraceElements: Array<StackTraceElement?>?) {
        stackTrace = stackTraceElements
    }
}